### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by musvaage**
https://github.com/zenodo/zenodo/issues/2423

### CODE REVIEW

**Comment by musvaage** [comment]
@mitsosf 

ping

**Review by ppanero — APPROVED** [review]

**Comment by musvaage** [comment]
> if you fix it

I'm not likely to do that.

**Review by mitsosf — APPROVED** [review]
Further elaborating on what @ppanero mentioned, I did some digging and the issue seems to be coming from there:

https://github.com/inveniosoftware/invenio-deposit/blob/master/invenio_deposit/jsonschemas/deposits/deposit-v1.0.0.json#L35
and this
https://github.com/inveniosoftware/invenio-deposit/blob/master/invenio_deposit/jsonschemas/deposits/deposit-v1.0.0.json#L40

Let's talk about how to tackle this tomorrow, maybe it's not worth it to include this specific word change but will need to look into the complexity a bit.

**Comment by musvaage** [comment]
"type" and "value"

```patch
diff --git a/zenodo/modules/deposit/jsonschemas/deposits/records/record-v1.0.0.json b/zenodo/modules/deposit/jsonschemas/deposits/records/record-v1.0.0.json
index 2b4cd5b20..a2892efad 100644
--- a/zenodo/modules/deposit/jsonschemas/deposits/records/record-v1.0.0.json
+++ b/zenodo/modules/deposit/jsonschemas/deposits/records/record-v1.0.0.json
@@ -233,12 +233,12 @@
               "type": "integer"
             },
             "type": {
-              "description": "Type of the PID, in correspondece with PID's 'pid_type' property.",
+              "description": "Type of the PID, in correspondence with PID's 'pid_type' property.",
               "title": "PID Type",
               "type": "string"
             },
             "value": {
-              "description": "Value of the PID, in correspondece with PID's 'pid_value' property.",
+              "description": "Value of the PID, in correspondence with PID's 'pid_value' property.",
               "title": "PID Value",
               "type": "string"
             }
```

```patch
diff --git a/zenodo/modules/records/jsonschemas/records/record-v1.0.0.json b/zenodo/modules/records/jsonschemas/records/record-v1.0.0.json
index 16da46461..ca8d5d010 100644
--- a/zenodo/modules/records/jsonschemas/records/record-v1.0.0.json
+++ b/zenodo/modules/records/jsonschemas/records/record-v1.0.0.json
@@ -232,12 +232,12 @@
               "type": "integer"
             },
             "type": {
-              "description": "Type of the PID, in correspondece with PID's 'pid_type' property.",
+              "description": "Type of the PID, in correspondence with PID's 'pid_type' property.",
               "title": "PID Type",
               "type": "string"
             },
             "value": {
-              "description": "Value of the PID, in correspondece with PID's 'pid_value' property.",
+              "description": "Value of the PID, in correspondence with PID's 'pid_value' property.",
               "title": "PID Value",
               "type": "string"
             }
```

File: zenodo/modules/deposit/jsonschemas/deposits/records/record-v1.0.0.json [hunk 1]
```diff
@@ -233,12 +233,12 @@
               "type": "integer"
             },
             "type": {
-              "description": "Type of the PID, in correspondece with PID's 'pid_type' property.",
+              "description": "Type of the PID, in correspondence with PID's 'pid_type' property.",
```
Comment (mitsosf) [inline zenodo/modules/deposit/jsonschemas/deposits/records/record-v1.0.0.json:236]:
```suggestion
              "description": "Type of the PID, in correspondece with PID's 'pid_type' property.",
```

File: zenodo/modules/deposit/jsonschemas/deposits/records/record-v1.0.0.json [hunk 2]
```diff
@@ -233,12 +233,12 @@
...
               "title": "PID Type",
               "type": "string"
             },
             "value": {
-              "description": "Value of the PID, in correspondece with PID's 'pid_value' property.",
+              "description": "Value of the PID, in correspondence with PID's 'pid_value' property.",
```
Comment (mitsosf) [inline zenodo/modules/deposit/jsonschemas/deposits/records/record-v1.0.0.json:241]:
```suggestion
              "description": "Value of the PID, in correspondece with PID's 'pid_value' property.",
```

File: zenodo/modules/records/jsonschemas/records/record-v1.0.0.json [hunk 3]
```diff
@@ -232,12 +232,12 @@
               "type": "integer"
             },
             "type": {
-              "description": "Type of the PID, in correspondece with PID's 'pid_type' property.",
+              "description": "Type of the PID, in correspondence with PID's 'pid_type' property.",
```
Comment (mitsosf) [inline zenodo/modules/records/jsonschemas/records/record-v1.0.0.json:235]:
```suggestion
              "description": "Type of the PID, in correspondece with PID's 'pid_type' property.",
```

**Review by mitsosf — CHANGES_REQUESTED** [review]
I think the easiest way to resolve this, for now, is to just revert this specific typo to its wrong form since the `invenio_deposit` package will eventually become redundant and we have other ongoing efforts at the moment.

File: zenodo/modules/deposit/jsonschemas/deposits/records/record-v1.0.0.json (hunk 1 above)
Comment (musvaage) [reply zenodo/modules/deposit/jsonschemas/deposits/records/record-v1.0.0.json]: uncommented the two relevant sed lines in my ticket's shell script

**Review by mitsosf — APPROVED** [review]

**Comment by musvaage** [comment]
Another typo exposed in `#2432`.

**Comment by musvaage** [comment]
If obligatory on the Developers' end can you rebase?

**Comment by ppanero** [comment]
@musvaage we cannot rebase because the PR has been made from your master branch and we cannot force push to it. Otherwise, we would do it :/ When doing PRs if you want to allow maintainers to push fixes you need to create them from a custom branch, not main/master.

**Comment by musvaage** [comment]
rebased

@mitsosf 

Can you approve running the workflow again?

**Pull request merged**